src/containers/OrdinalsArea.js

```diff
@@ -72,19 +51,76 @@ const OrdinalsArea = ({ className, space }) => {
         setRefreshHack(!refreshHack);
     };
 
+    const getOutpointFromCache = async (inscriptionId) => {
+        const key = `${LocalStorageKeys.INSCRIPTIONS_OUTPOINT}:${inscriptionId}`;
+        const cachedOutpoint = await LocalStorage.get(key);
+        if (cachedOutpoint) {
+            return cachedOutpoint;
+        }
+
+        const {
+            data: {
+                inscription: { outpoint },
+            },
+        } = await axios.get(`https://turbo.ordinalswallet.com/inscription/${inscriptionId}/outpoint`);
+
+        await LocalStorage.set(key, outpoint);
+
+        return outpoint;
+    };
+
     useEffect(() => {
-        const fetchByUtxos = async () => {
+        const loadUtxos = async () => {
             setUtxosReady(false);
-            const ownedInscriptions = await getOwnedInscriptions(nostrAddress);
-            setOwnedUtxos(ownedInscriptions);
-            setFilteredOwnedUtxos(ownedInscriptions);
-            const ownedInscriptionResults = await Promise.allSettled(
-                ownedInscriptions.map((utxo) => getInscriptionData(utxo))
-            );
-            setOwnedUtxos(ownedInscriptionResults.map((utxo) => utxo.value));
+
+            const utxos = await getSortedUtxos(nostrAddress);
+            const inscriptions = await axios.get(`${TURBO_API}/wallet/${nostrAddress}/inscriptions`);
+
+            const inscriptionsByUtxoKey = {};
+            const batchPromises = [];
+            const populateInscriptionsMap = async (ins) => {
+                const outpoint = await getOutpointFromCache(ins.id);
```

Comment:
@andredevjs sorry 1 question, couldn't getOutpointFromCache be replaced with an await.fetch() and then utilize 'force-cache' mechanism to cache it server side? Then all users can benefit

Also maybe dumb question, but why do we sometimes use fetch and sometimes axios?